src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);

```

Comment:
```suggestion
public class ColorSupplier {
    private final Random random = new Random();
    
    public String getRandomColor() {
        int index = random.nextInt(Color.values().length);
```
Random should be a class-variable